pcdsdevices/lxe.py

```diff
@@ -359,6 +361,69 @@ def set_current_position(self, position):
         new_offset = position - self.setpoint.get()
         self.user_offset.put(new_offset)
 
+    @property
+    def verbose_name(self):
+        """Get the verbose name for LXT motor"""
+        return self._lxt_verbose_name
+
+    @verbose_name.setter
+    def verbose_name(self, name):
+        """
+        Set a verbose name for the LXT motor.
+
+        Parameters
+        ----------
+        name : str
+            Verbose name for the LXT motor.
+        """
+        self._lxt_verbose_name = name
```

Comment:
If you have a property that just gets/sets an internal variable with no additional logic, wrapping it with a property is unnecessary. Would suggest just renaming `_lxt_verbose_name` to `verbose_name` and leaving it at that.

Reply:
Yeah that make sense, thanks Ken, removed these guys in 75a6f41

Also wrapped getting the `dial_pos` in a try/except - maybe it is not the best place to wrap the in a try/except, but if we only use it to format the status for it I think it should be fine.

Reply:
No problem - and that seems a reasonable place for the try/except 👍

Reply:
One situation where I think it makes sense to add these sorts of flat properties is when you need to document something and it makes a lot of sense to do it at the property. This sort of documentation is easier to follow then skimming through long class docstrings or doing it inline with comments.